src/laion_clap/clap_module/factory.py

```diff
@@ -5,6 +5,7 @@
 import re
 from copy import deepcopy
 from pathlib import Path
+from packaging import version
```

Comment:
Since it is already a dependency of setup tools - we don't quite need to install this.